web/src/beta/features/Editor/tabs/story/LeftPanel/ContentPage/index.tsx

```diff
@@ -60,6 +60,7 @@ const ContentPage: React.FC<Props> = ({
           renderItem={(storyPage, i) => {
             const title = (getFieldValue(storyPage.property.items ?? [], "title", "title") ??
               t("Untitled")) as string;
+            const hasEmptySpace = /^ *$/.test(title);
```

Comment:
Can you move this in to a global util so we can use it elsewhere. Something like [this](https://stackoverflow.com/questions/57476487/how-to-check-if-a-string-contains-only-white-space-or-is-empty-in-typescript#:~:text=export%20function%20isEmpty(text%3A%20string)%3A%20boolean%20%7B%0A%20%20return%20text%20%3D%3D%20null%20%7C%7C%20text.match(/%5E%5Cs*%24/)%20!%3D%3D%20null%3B%0A%7D) might be more useable? Maybe

Reply:
done